### PR TITLE
MM 27448 remove jquery from root

### PR DESCRIPTION
**Pull request by CEOehis**
#### Summary

This PR removes the usage of jQuery from the following files in the web app:

components/logged_in/logged_in.jsx
components/root/root.jsx
components/tutorial/tutorial_view.jsx
#### Ticket Link

Fixes https://mattermost.atlassian.net/browse/MM-27448
Fixes https://github.com/mattermost/mattermost-server/issues/15159

### CODE REVIEW

**Comment by hanzei** [comment]
Awesome, great to see more removal of jquery. Heads up that one test is failing with:
```
Summary of all failing tests
 FAIL  components/logged_in/logged_in.test.jsx
  ● components/logged_in/LoggedIn › should render loading state without user

    TypeError: Cannot read property 'setAttribute' of null

      83 | 
      84 |         if (!this.props.currentUser) {
    > 85 |             document.getElementById('root').setAttribute('class', '');
         |             ^
      86 |             GlobalActions.emitUserLoggedOutEvent('/login?redirect_to=' + encodeURIComponent(this.props.location.pathname), true, false);
      87 |         }
      88 | 

      at LoggedIn.componentDidMount (components/logged_in/logged_in.jsx:85:13)
      at fn (node_modules/enzyme/src/ShallowWrapper.js:429:22)
      at Object.batchedUpdates (node_modules/enzyme-adapter-react-16/src/ReactSixteenAdapter.js:771:16)
      at new ShallowWrapper (node_modules/enzyme/src/ShallowWrapper.js:428:26)
      at shallow (node_modules/enzyme/src/shallow.js:10:10)
      at Object.it (components/logged_in/logged_in.test.jsx:38:25)
```

**Review by hmhealey — CHANGES_REQUESTED** [review]

**Review by nickmisasi — APPROVED** [review]
Sorry for the delay. Everything looks good to me! Thanks @CEOehis!

File: e2e/cypress/integration/keyboard_shortcuts/backspace_spec.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+        cy.visit(`/${testTeam.name}/channels/off-topic`);
+        cy.visit(`/${testTeam.name}/channels/${publicChannel.name}`);
+        cy.visit(`/${testTeam.name}/channels/off-topic`);
+
+        cy.visit(`/${testTeam.name}/messages/@${testUser.username}`);
+        cy.url().should('include', `/${testTeam.name}/messages/@${testUser.username}`);
+
+        // click on the body to remove focus from the input key
+        cy.get('#post_textbox').clear().type('This is a normal sentence.').type('{backspace}{backspace}').blur();
```
Comment (josephbaylon) [inline e2e/cypress/integration/keyboard_shortcuts/backspace_spec.js:40]: Please add a verification that the last 2 characters of the message have been removed.

**Review by josephbaylon — CHANGES_REQUESTED** [review]
@CEOehis Thanks for your contribution!! Just a couple of minor request changes on the E2E.

File: e2e/cypress/integration/keyboard_shortcuts/backspace_spec.js (hunk 1 above)
Comment (josephbaylon) [reply e2e/cypress/integration/keyboard_shortcuts/backspace_spec.js:40]: Also after the focus has been removed from post input, add another verification that no further truncation happened to the message in the input.

**Review by josephbaylon — APPROVED** [review]
LGTM. Test passed locally. Thanks @CEOehis !!

**Comment by hanzei** [comment]
Thanks for the awesome contribution @CEOehis :+1:

**Pull request merged**